repository/flink/docs/demo/financial-fraud/README.md

```diff
@@ -0,0 +1,127 @@
+# Flink Financial Fraud Demo
+
+This demo follows the outline provided by [DCOS's](https://github.com/dcos/demos/tree/master/flink-k8s/1.11) demo
```

Comment:
There is an updated version of this demo whose URL is https://github.com/dcos/demos/tree/master/flink-k8s/1.13

Reply:
Looking over https://github.com/dcos/demos/pull/86 it seems there are overall minor wording changes. Was there anything in particular note worthy that we should add in this demo here as well?

Reply:
I think @gkleiman caught most of - and made similar recommendations to - anything wording related.